packages/fiori/test/pages/ViewSettingsDialog.html

```diff
@@ -97,6 +100,9 @@ <h3> ViewSettingsDialog</h3>
 		btnOpenDialog.addEventListener("click", function () {
 			vsd.show();
 		});
+		btnChangeSettings.addEventListener("click", function () {
```

Comment:
What's the reasoning behind listening to a `click` event rather than `before-open` as mentioned in the `setConfirmedSettings` description here: https://github.com/SAP/ui5-webcomponents/pull/5222/files#diff-aecdde634565d64ffb6f8be2c447e63c14bca4c9949574f24aa86e0c8480d8bbR683?

On a similar note, should this event listener be after line `100`?